task/power/src/main.rs

```diff
@@ -228,17 +234,64 @@ fn controllers() -> [PowerController; 15] {
     ]
 }
 
+#[cfg(any(target_board = "gimlet-a", target_board = "gimlet-b"))]
```

Comment:
Vague (non-actionable) feelings: It seems weird that we can either check `target_board` *or* the `gimlet` feature when deciding what to enable here.